website/project/model.py

```diff
@@ -4004,16 +4008,51 @@ def approve(self, user):
         self.approval.approve(user)
         self.add_status_log(user, DraftRegistrationLog.APPROVED)
         self.approval.save()
+        self.checkin_files(save=True)
```

Comment:
If multiple users need to `.approve` a registration before it goes through, could this potentially trigger a check-in too early?